example/tools/BUILD

```diff
@@ -132,6 +141,7 @@ shellcheck_binary(name = "shellcheck")
 multi_formatter_binary(
     name = "format",
     cc = "@llvm_toolchain_llvm//:bin/clang-format",
+    # Or ":gofumpt" if you want to use gofumpt instead of gofmt
```

Comment:
it's a bit rough that we have no way to test gofumpt is working at all. Maybe we need another `multi_formatter_binary` in the test folder.

Reply:
I can add something like this:

```
multi_formatter_binary(
    name = "format_fumpt",
    go = ":gofumpt"
)
```

is that actually going to be exercised by anything though?

Reply:
No, and I don't see any test fixture we could extend under the example/ folder either. This is fine already, it's so simple there's not much surface area where it could have a bug.